type_inference_visitor.py

```diff
@@ -0,0 +1,126 @@
+from astroid.transforms import TransformVisitor
+import astroid
+import warnings
+from typing import Tuple, List, Dict
+
+
+class TypeVisitor(TransformVisitor):
+
+    def _visit(self, node):
```

Comment:
I found that we don't need to override anything in TransfromVistor.. should I just use TransfromVisitor instead of TypeVisitor?